diff --git a/node_modules/@mui/x-date-pickers/CalendarPicker/CalendarPicker.js b/node_modules/@mui/x-date-pickers/CalendarPicker/CalendarPicker.js
index 4803d41..d414ab8 100644
--- a/node_modules/@mui/x-date-pickers/CalendarPicker/CalendarPicker.js
+++ b/node_modules/@mui/x-date-pickers/CalendarPicker/CalendarPicker.js
@@ -5,7 +5,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { useControlled, unstable_useId as useId, useEventCallback } from '@mui/material/utils';
 import { MonthPicker } from '../MonthPicker/MonthPicker';
 import { useCalendarState } from './useCalendarState';
diff --git a/node_modules/@mui/x-date-pickers/CalendarPicker/DayPicker.js b/node_modules/@mui/x-date-pickers/CalendarPicker/DayPicker.js
index 702cc85..3f2a96f 100644
--- a/node_modules/@mui/x-date-pickers/CalendarPicker/DayPicker.js
+++ b/node_modules/@mui/x-date-pickers/CalendarPicker/DayPicker.js
@@ -2,7 +2,7 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import { styled, useTheme, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import clsx from 'clsx';
 import { PickersDay } from '../PickersDay/PickersDay';
 import { useUtils, useNow } from '../internals/hooks/useUtils';
diff --git a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersCalendarHeader.js b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersCalendarHeader.js
index d9b7ea0..8fdd70f 100644
--- a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersCalendarHeader.js
+++ b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersCalendarHeader.js
@@ -2,7 +2,7 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 import * as React from 'react';
 import Fade from '@mui/material/Fade';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import IconButton from '@mui/material/IconButton';
 import { useLocaleText, useUtils } from '../internals/hooks/useUtils';
 import { PickersFadeTransitionGroup } from './PickersFadeTransitionGroup';
diff --git a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersFadeTransitionGroup.js b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersFadeTransitionGroup.js
index e1f1ed7..accb400 100644
--- a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersFadeTransitionGroup.js
+++ b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersFadeTransitionGroup.js
@@ -2,7 +2,7 @@ import * as React from 'react';
 import clsx from 'clsx';
 import Fade from '@mui/material/Fade';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { TransitionGroup } from 'react-transition-group';
 import { getPickersFadeTransitionGroupUtilityClass } from './pickersFadeTransitionGroupClasses';
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersSlideTransition.js b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersSlideTransition.js
index 2ec7424..10f21bf 100644
--- a/node_modules/@mui/x-date-pickers/CalendarPicker/PickersSlideTransition.js
+++ b/node_modules/@mui/x-date-pickers/CalendarPicker/PickersSlideTransition.js
@@ -4,7 +4,7 @@ const _excluded = ["children", "className", "reduceAnimations", "slideDirection"
 import * as React from 'react';
 import clsx from 'clsx';
 import { styled } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { CSSTransition, TransitionGroup } from 'react-transition-group';
 import { getPickersSlideTransitionUtilityClass, pickersSlideTransitionClasses } from './pickersSlideTransitionClasses';
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@mui/x-date-pickers/ClockPicker/ClockNumber.js b/node_modules/@mui/x-date-pickers/ClockPicker/ClockNumber.js
index eb0b20c..8c3a536 100644
--- a/node_modules/@mui/x-date-pickers/ClockPicker/ClockNumber.js
+++ b/node_modules/@mui/x-date-pickers/ClockPicker/ClockNumber.js
@@ -4,7 +4,7 @@ const _excluded = ["className", "disabled", "index", "inner", "label", "selected
 import * as React from 'react';
 import clsx from 'clsx';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { CLOCK_WIDTH, CLOCK_HOUR_WIDTH } from './shared';
 import { getClockNumberUtilityClass, clockNumberClasses } from './clockNumberClasses';
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@mui/x-date-pickers/ClockPicker/ClockPicker.js b/node_modules/@mui/x-date-pickers/ClockPicker/ClockPicker.js
index e8af2dd..ce1a385 100644
--- a/node_modules/@mui/x-date-pickers/ClockPicker/ClockPicker.js
+++ b/node_modules/@mui/x-date-pickers/ClockPicker/ClockPicker.js
@@ -4,7 +4,7 @@ import clsx from 'clsx';
 import PropTypes from 'prop-types';
 import { unstable_useId as useId } from '@mui/material/utils';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { Clock } from './Clock';
 import { useUtils, useNow, useLocaleText } from '../internals/hooks/useUtils';
 import { buildDeprecatedPropsWarning } from '../internals/utils/warning';
diff --git a/node_modules/@mui/x-date-pickers/ClockPicker/ClockPointer.js b/node_modules/@mui/x-date-pickers/ClockPicker/ClockPointer.js
index 53e548e..9e59b7a 100644
--- a/node_modules/@mui/x-date-pickers/ClockPicker/ClockPointer.js
+++ b/node_modules/@mui/x-date-pickers/ClockPicker/ClockPointer.js
@@ -4,7 +4,7 @@ const _excluded = ["className", "hasSelected", "isInner", "type", "value"];
 import * as React from 'react';
 import clsx from 'clsx';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { CLOCK_WIDTH, CLOCK_HOUR_WIDTH } from './shared';
 import { getClockPointerUtilityClass } from './clockPointerClasses';
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@mui/x-date-pickers/DatePicker/DatePickerToolbar.js b/node_modules/@mui/x-date-pickers/DatePicker/DatePickerToolbar.js
index 88bacb1..910c287 100644
--- a/node_modules/@mui/x-date-pickers/DatePicker/DatePickerToolbar.js
+++ b/node_modules/@mui/x-date-pickers/DatePicker/DatePickerToolbar.js
@@ -4,7 +4,7 @@ const _excluded = ["parsedValue", "isLandscape", "isMobileKeyboardViewOpen", "on
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { PickersToolbar } from '../internals/components/PickersToolbar';
 import { useLocaleText, useUtils } from '../internals/hooks/useUtils';
 import { isYearAndMonthViews, isYearOnlyView } from './shared';
diff --git a/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerTabs.js b/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerTabs.js
index d8c547e..ff379e7 100644
--- a/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerTabs.js
+++ b/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerTabs.js
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import Tab from '@mui/material/Tab';
 import Tabs, { tabsClasses } from '@mui/material/Tabs';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { Time, DateRange } from '../internals/components/icons';
 import { WrapperVariantContext } from '../internals/components/wrappers/WrapperVariantContext';
 import { useLocaleText } from '../internals/hooks/useUtils';
diff --git a/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerToolbar.js b/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerToolbar.js
index aeab092..b82ef67 100644
--- a/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerToolbar.js
+++ b/node_modules/@mui/x-date-pickers/DateTimePicker/DateTimePickerToolbar.js
@@ -3,7 +3,7 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 const _excluded = ["ampm", "parsedValue", "isMobileKeyboardViewOpen", "onChange", "openView", "setOpenView", "toggleMobileKeyboardView", "toolbarFormat", "toolbarPlaceholder", "toolbarTitle", "views"];
 import * as React from 'react';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { PickersToolbarText } from '../internals/components/PickersToolbarText';
 import { PickersToolbar } from '../internals/components/PickersToolbar';
 import { pickersToolbarClasses } from '../internals/components/pickersToolbarClasses';
diff --git a/node_modules/@mui/x-date-pickers/MonthPicker/MonthPicker.js b/node_modules/@mui/x-date-pickers/MonthPicker/MonthPicker.js
index 9d30e97..7481be2 100644
--- a/node_modules/@mui/x-date-pickers/MonthPicker/MonthPicker.js
+++ b/node_modules/@mui/x-date-pickers/MonthPicker/MonthPicker.js
@@ -6,7 +6,9 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { useTheme } from '@mui/system';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses, useControlled, useEventCallback } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
+import useControlled from '@mui/utils/useControlled'
+import useEventCallback from '@mui/utils/useEventCallback'
 import { PickersMonth } from './PickersMonth';
 import { useUtils, useNow, useDefaultDates } from '../internals/hooks/useUtils';
 import { getMonthPickerUtilityClass } from './monthPickerClasses';
diff --git a/node_modules/@mui/x-date-pickers/MonthPicker/PickersMonth.js b/node_modules/@mui/x-date-pickers/MonthPicker/PickersMonth.js
index 0b953e0..676b029 100644
--- a/node_modules/@mui/x-date-pickers/MonthPicker/PickersMonth.js
+++ b/node_modules/@mui/x-date-pickers/MonthPicker/PickersMonth.js
@@ -3,8 +3,9 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 const _excluded = ["disabled", "onSelect", "selected", "value", "tabIndex", "hasFocus", "onFocus", "onBlur"];
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
-import { styled, alpha } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { alpha } from '@mui/system/colorManipulator';
+import composeClasses from '@mui/utils/composeClasses';
 import { unstable_useEnhancedEffect as useEnhancedEffect } from '@mui/material/utils';
 import { onSpaceOrEnter } from '../internals/utils/utils';
 import { getPickersMonthUtilityClass, pickersMonthClasses } from './pickersMonthClasses';
diff --git a/node_modules/@mui/x-date-pickers/PickersDay/PickersDay.js b/node_modules/@mui/x-date-pickers/PickersDay/PickersDay.js
index 5cf5f01..88f672d 100644
--- a/node_modules/@mui/x-date-pickers/PickersDay/PickersDay.js
+++ b/node_modules/@mui/x-date-pickers/PickersDay/PickersDay.js
@@ -6,8 +6,9 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import ButtonBase from '@mui/material/ButtonBase';
 import { unstable_useEnhancedEffect as useEnhancedEffect } from '@mui/utils';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
-import { alpha, styled, useThemeProps } from '@mui/material/styles';
+import composeClasses from '@mui/utils/composeClasses';
+import { styled, useThemeProps } from '@mui/material/styles';
+import { alpha } from '@mui/system/colorManipulator';
 import { useForkRef } from '@mui/material/utils';
 import { useUtils } from '../internals/hooks/useUtils';
 import { DAY_SIZE, DAY_MARGIN } from '../internals/constants/dimensions';
diff --git a/node_modules/@mui/x-date-pickers/YearPicker/PickersYear.js b/node_modules/@mui/x-date-pickers/YearPicker/PickersYear.js
index 17fb1de..395e6f3 100644
--- a/node_modules/@mui/x-date-pickers/YearPicker/PickersYear.js
+++ b/node_modules/@mui/x-date-pickers/YearPicker/PickersYear.js
@@ -4,8 +4,9 @@ const _excluded = ["autoFocus", "className", "children", "disabled", "onClick",
 import * as React from 'react';
 import clsx from 'clsx';
 import { useForkRef, capitalize } from '@mui/material/utils';
-import { alpha, styled } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { alpha } from '@mui/system/colorManipulator';
+import composeClasses from '@mui/utils/composeClasses';
 import { WrapperVariantContext } from '../internals/components/wrappers/WrapperVariantContext';
 import { getPickersYearUtilityClass, pickersYearClasses } from './pickersYearClasses';
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@mui/x-date-pickers/YearPicker/YearPicker.js b/node_modules/@mui/x-date-pickers/YearPicker/YearPicker.js
index 6b4ee84..d40e339 100644
--- a/node_modules/@mui/x-date-pickers/YearPicker/YearPicker.js
+++ b/node_modules/@mui/x-date-pickers/YearPicker/YearPicker.js
@@ -2,7 +2,7 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { useTheme, styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import clsx from 'clsx';
 import { useForkRef } from '@mui/material/utils';
 import { unstable_useControlled as useControlled } from '@mui/utils';
diff --git a/node_modules/@mui/x-date-pickers/internals/components/CalendarOrClockPicker/CalendarOrClockPicker.js b/node_modules/@mui/x-date-pickers/internals/components/CalendarOrClockPicker/CalendarOrClockPicker.js
index de5ef38..8cc7893 100644
--- a/node_modules/@mui/x-date-pickers/internals/components/CalendarOrClockPicker/CalendarOrClockPicker.js
+++ b/node_modules/@mui/x-date-pickers/internals/components/CalendarOrClockPicker/CalendarOrClockPicker.js
@@ -3,7 +3,7 @@ import _extends from "@babel/runtime/helpers/esm/extends";
 const _excluded = ["autoFocus", "className", "parsedValue", "DateInputProps", "isMobileKeyboardViewOpen", "onDateChange", "onViewChange", "openTo", "orientation", "showToolbar", "toggleMobileKeyboardView", "ToolbarComponent", "toolbarFormat", "toolbarPlaceholder", "toolbarTitle", "views", "dateRangeIcon", "timeIcon", "hideTabs", "classes"];
 import * as React from 'react';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import { useViews } from '../../hooks/useViews';
 import { ClockPicker } from '../../../ClockPicker/ClockPicker';
 import { CalendarPicker } from '../../../CalendarPicker/CalendarPicker';
diff --git a/node_modules/@mui/x-date-pickers/internals/components/PickersArrowSwitcher.js b/node_modules/@mui/x-date-pickers/internals/components/PickersArrowSwitcher.js
index 04a567f..5d32fe8 100644
--- a/node_modules/@mui/x-date-pickers/internals/components/PickersArrowSwitcher.js
+++ b/node_modules/@mui/x-date-pickers/internals/components/PickersArrowSwitcher.js
@@ -5,7 +5,7 @@ import * as React from 'react';
 import clsx from 'clsx';
 import Typography from '@mui/material/Typography';
 import { useTheme, styled, useThemeProps } from '@mui/material/styles';
-import { unstable_composeClasses as composeClasses } from '@mui/material';
+import composeClasses from '@mui/utils/composeClasses';
 import IconButton from '@mui/material/IconButton';
 import { ArrowLeft, ArrowRight } from './icons';
 import { getPickersArrowSwitcherUtilityClass } from './pickersArrowSwitcherClasses';